### ISSUE TITLE
(🐞) `unused-ignore` doesn't activate on disabled error codes

### ISSUE THREAD

**Issue by KotlinIsland** [python/mypy#16537]
```py
# mypy: enable-error-code=unused-ignore, disable-error-code=operator
1  + ""  # type: ignore
# no error reported
```

The expected output here would be:
```
error: Unused "type: ignore" comment  [unused-ignore]
```

- related #16536
